src/wp-admin/includes/class-wp-automatic-updater.php

```diff
@@ -1017,7 +1083,16 @@ protected function send_plugin_theme_email( $type, $successful_updates, $failed_
 				$body[] = __( 'These plugins failed to update:' );
 
 				foreach ( $failed_updates['plugin'] as $item ) {
-					$body[] = "- {$item->name}";
+					$body[]                                = "- {$item->name}";
+					
+					// reset the plugin time if time has been passed
+					if ( isset($failure_emails[ $item->item->plugin ] ) ) {
+						if ( time() - $failure_emails[ $item->item->plugin ] > $failure_email_interval ) {
+							$failure_emails[ $item->item->plugin ] = time();
+						}
+					} else {
+						$failure_emails[ $item->item->plugin ] = time();
+					}
```

Comment:
Reset the plugin or theme time if time has been passed for 3 days or more

Reply:
This extra logic is not needed, the one line that was there is sufficient.